### ISSUE TITLE
Form model and image field

### ISSUE THREAD

**Issue by JuliaTyumen** [kristijanhusak/laravel-form-builder#87]
Thank you for your useful package! But I have some problems with using image field type and using model. I tried to use field customization (to create my field template), but model always use attribute "value" for it's values. For example, when I tried this code in view file:
```
<?= Form::image(null, $name) ?>
```
I got that output code:
```
<input src="http://localhost/laravelapps/prohunting/public/" name="picture" type="image" value="IMG_0332.png" id="picture">
```
Value IMG_0332.png was taken from model and it didn't appear in src attribute.
Is there any way to use model with image field?

**Comment by kristijanhusak** [comment]
If you check [here](https://github.com/illuminate/html/blob/master/FormBuilder.php#L698), it needs first param, which you set to `null`, to be the source of the image. I also didn't notice that, so i need to fix when using default `image` field type, but you just pass the `src` for your image as first attribute

**Comment by JuliaTyumen** [comment]
@kristijanhusak thanks for your reply. But I didn't understant where and how can I get image source value from model for this param? It's unnavailable in blade file (array $options doesn't have it). Can I get it in class App\Forms\Fields\ImageType? Right now my class for cstom field just like in the documentation:
```
<?php namespace App\Forms\Fields;

use Kris\LaravelFormBuilder\Fields\FormField;

class ImageType extends FormField {

    protected function getTemplate()
    {
        return 'fields.image';
    }

    public function render(array $options = [], $showLabel = true, $showField = true, $showError = true)
    {
        $options['somedata'] = 'This is some data for view';

        return parent::render($options, $showLabel, $showField, $showError);
    }
}
```
Can i get data from model there?

**Comment by kristijanhusak** [comment]
Do you need `<input type="image">` or you just need regular image `<img src="">` ?

**Comment by JuliaTyumen** [comment]
Oooh, I'm sorry, I was wrong. I need to show thumbnail image if there is image source in model, and there should be button "Change image" - it should be file field.
So I tried to use static field for image, and file field for new image, and it worked (I changed static.blade.php).
So if there will be new file after form submit I'll save it to the model (change picture).

Thank you very much for your help!
